sentinel-ai-core/src/main/java/com/phonepe/sentinelai/core/earlytermination/EarlyTerminationStrategyResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.phonepe.sentinelai.core.earlytermination;
+
+import com.phonepe.sentinelai.core.errors.ErrorType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Better to remove both of these, and the builder. Mark with `@AllArgsConstructor(access=PRIVATE)` and provlide static `doNotTerminate()` and `terminate()` methods to create objects.